packages/lite-insight/src/insights/extractors/changePoint.ts

```diff
@@ -1,36 +1,19 @@
-import { BayesianChangePoint, BreakPoint, pettittPValue } from '../../algorithms';
+import { ChangePoint } from '../../algorithms';
 import { SignificanceBenchmark } from '../../constant';
 import { Datum, ChangePointInfo } from '../../interface';
 
-const breakpointVerifier = (
-  next: BreakPoint<number>,
-  prev: BreakPoint<number>
-): boolean => {
-  if (Math.abs(next.data - prev.data) >= 1) {
-    return true;
-  }
-
-  return false;
-};
-
 type ChangePointItem = {
   index: number;
   significance: number;
 };
 
 export const findChangePoints = (series: number[]): ChangePointItem[] => {
-  const detection = new BayesianChangePoint<number>({
-    breakpointVerifier
-  });
-
-  detection.exec(series);
-
-  const testResult = detection.breakPoints().map(breakPoint => ( { index: breakPoint.index, significance: pettittPValue(series, breakPoint.index) }));
+  const result = ChangePoint.Bayesian(series);
```

Comment:
Why use plural `breakdownValues` in `packages/lite-insight/src/pipeline/insight.ts`, `schemas` in `packages/lite-insight/src/visualization/index.ts`, but singular `result` here.

https://github.com/antvis/AVA/pull/283/files#diff-625a884578cf1bb31d6590d9baec631e24d31a72599da8ccf7348454357ba007R152

https://github.com/antvis/AVA/pull/283/files#diff-e3822dbeeae836666bcecbc5d5769e07c278824b2461612dfa41db84b426b7d5R17